### PR TITLE
DO NOT MERGE: Feature/controller and op bindings

### PR DESCRIPTION
**Pull request by ritch**
WIP

### CODE REVIEW

File: packages/loopback/lib/application.ts [hunk 1]
```diff
@@ -19,8 +19,10 @@ export class Application extends Context {
         throw new Error(`The controller ${b.key} was not bound via .toClass()`);
       }
 
-      const ctorFactory = (req: http.ServerRequest, res: http.ServerResponse) => {
+      const ctorFactory = (req: http.ServerRequest, res: http.ServerResponse, operationName: string) => {
         const requestContext = new Context(this);
+        requestContext.bind('controller.current.ctor').to(ctor);
```
Comment (bajtos) [inline packages/loopback/lib/application.ts]: nitpick: can we avoid abbreviations? `constroller.current.constructor`.

File: packages/authentication/test/acceptance/authentication/feature.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 ## Scenario: Basic Usage
 
 ```ts
+////////// acceptance
```
Comment (superkhau) [inline packages/authentication/test/acceptance/authentication/feature.md]: remove

File: packages/loopback/lib/application.ts (hunk 1 above)
Comment (superkhau) [inline packages/loopback/lib/application.ts]: `controller`

File: packages/loopback/test/acceptance/routing/routing.acceptance.ts [hunk 3]
```diff
@@ -161,6 +161,38 @@ describe('Routing', () => {
     });
   });
 
+    it('binds controller constructor object and operation', async () => {
```
Comment (superkhau) [inline packages/loopback/test/acceptance/routing/routing.acceptance.ts]: binds the controller constructor object with the given operation

**Pull request merged**